app/_hub/kong-inc/serverless-functions/index.md

```diff
@@ -103,7 +103,7 @@ different priority in the plugin chain.
     ...
     ```
 
-6. Test that our lua code will terminate the request when no header is passed:
```

Comment:
@hishamhm @thibaultcha do you prefer `lua` or `Lua`?

Reply:
Please use `Lua`.

Reply:
Also see the "What's in a name?" section at https://www.lua.org/about.html